wallet/controllers_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
are we adding the new 1.17 directive

Reply:
must be my vim-go doing it automatically... I think as long as you have both it will work cross version.